src/commands/org/create/sandbox.ts

```diff
@@ -79,18 +79,27 @@ export default class CreateSandbox extends SandboxCommandBase<SandboxCommandResp
         return Promise.resolve(name);
       },
     }),
-    clone: Flags.string({
+    'source-sandbox-name': Flags.string({
       char: 'c',
-      summary: messages.getMessage('flags.clone.summary'),
-      description: messages.getMessage('flags.clone.description'),
+      summary: messages.getMessage('flags.source-sandbox-name.summary'),
+      description: messages.getMessage('flags.source-sandbox-name.description'),
+      exclusive: ['license-type', 'source-id'],
+      deprecateAliases: true,
+      aliases: ['clone', 'c'],
```

Comment:
`char: 'c'` on L83 should be removed (can't be a valid and deprecated short flag at the same time)